src/org/centos/contra/pipeline/Utils.groovy

```diff
@@ -319,7 +319,7 @@ def trackMessage(String messageID, int retryCount, def dataGrepperWebAddr=null)
         // We only want to wait if there are 404 errors
         echo "${STATUSCODE}"
         if (STATUSCODE.equals("404")) {
-            error("message not found on datagrepper...")
+            echo("message not found on datagrepper...")
```

Comment:
error() is a stopper, it exits the program. By replacing it with `echo`, the program continues, so you should be considerate about the code that follows. Currently, it will print "found!" and proceed.